doc/nrf/doc_build.rst

```diff
@@ -86,30 +86,42 @@ Complete the following steps to build the documentation output:
            ninja zephyr
 
    This step can take up to 15 minutes.
-#. Run ninja to build the nrfxlib documentation:
+
+#. Run ninja to build the mcuboot documentation:
 
         .. code-block:: console
 
-           ninja nrfxlib
+           ninja mcuboot
 
-#. Run ninja to build the mcuboot documentation:
+#. Run ninja to build the nrfxlib inventory file (used by nrf):
 
         .. code-block:: console
 
-           ninja mcuboot
+           ninja nrfxlib-inventory
 
 #. Run ninja to build the |NCS| documentation:
 
         .. code-block:: console
 
            ninja nrf
 
+#. Run ninja to build the nrfxlib documentation:
+
+        .. code-block:: console
+
+           ninja nrfxlib
+
 The documentation output is written to ``_build\html``. Double-click the ``index.html`` file to display the documentation in your browser.
 
 .. tip::
-   If you modify or add RST files, you only need to rerun the steps that build the respective documentation: step 4 (if you did changes to Kconfig), step 5 (if you modified the Zephyr documentation), step 6 (if you modified the nrfxlib documentation), step 7 (if you modified the MCUboot documentation), or step 8 (if you modified the |NCS| documentation).
 
+   If you modify or add RST files, you don't need to rerun all steps. For simple
```

Comment:
A bit more nitpicking, sorry: Please format with one sentence per line (according to our style guide), and don't use "don't". ;)

Reply:
Done!